glide.yaml

```diff
@@ -0,0 +1,14 @@
+package: github.com/yunify/qingcloud-sdk-go
+import:
+- package: github.com/DATA-DOG/godog
```

Comment:
I would like to take some time to refactor the test case after I resolve cni work item.